plugins/hls-eval-plugin/test/testdata/TInfo.expected.hs

```diff
@@ -0,0 +1,10 @@
+module TInfo (Eq, Ord, Foo) where
+
+import InfoUtil (Eq, Ord, Foo)
+
+-- >>> :info Foo
```

Comment:
I ended up changing these tests to work on a custom `Foo` (and `Bar`) datatype defined in `InfoUtil`